.github/workflows/release.yml

```diff
@@ -184,27 +184,33 @@ jobs:
       matrix:
         python-version: [3.6, 3.7, 3.8]
     steps:
+      - name: Configure Pagefile
+        uses: al-cheb/configure-pagefile-action@v1.2
+        with:
+          minimum-size: 4GB
+      - name: Pagefile size
+        run: |
+          (Get-CimInstance Win32_PageFileUsage).AllocatedBaseSize
+        shell: pwsh
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
       - name: Build Windows wheels
         run: |
-          export LCE_RELEASE_VERSION=${{ github.event.inputs.version }}
-          export PYTHON_BIN_PATH=$(which python)
-          export BAZEL_VC="C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/"
-
-          mkdir C:/bazel_cache
+          $Env:PYTHON_BIN_PATH = Get-Command python | Select-Object -ExpandProperty Definition
```

Comment:
Not sure if we still need this since we now set it already in #589

Reply:
Ah yeah you're right, they must resolve to the same thing :+1: